meson_cpu/meson.build

```diff
@@ -92,30 +92,27 @@ min_features = {
   'ppc64': [],
   's390x': [],
   'arm': [],
-  'aarch64': [ASIMD]
+  'aarch64': [ASIMD],
+  'wasm32': [],
 }.get(cpu_family, [])
 if host_machine.endian() == 'little' and cpu_family == 'ppc64'
   min_features = [VSX2]
 endif
 
-# Used by build option 'max/native/detect'
+# Used by build option 'max'
```

Comment:
I don't think this needed changing either?

Reply:
Will revert – I am unsure where this crept in from